touchforms/backend/decorators.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import with_statement
+
+
+def modify_xform_session(fn):
```

Comment:
if anyone has a better name for this decorator i'd welcome it. couldn't think of anything better and i'm not super happy with it. basically it allows you to edit an xform thread safe and it'll cache the xform once you're done modifying it